src/Spreadsheet/DeliverySpreadsheetParser.php

```diff
@@ -33,6 +38,7 @@ public function __construct(Geocoder $geocoder, PhoneNumberUtil $phoneNumberUtil
         $this->entityManager = $entityManager;
         $this->slugify = $slugify;
         $this->translator = $translator;
+        $this->defaultCoordinates = new GeoCoordinates(0,0);
```

Comment:
Maybe we should use the instance default coordinates instead? In this case, the `$defaultCoordinates` should not be initialized in the constructor, to avoid unnecessary database calls when the class is initialized by dependency injection.

Reply:
it makes more sense to me as well, so the dispatcher can see there is something wrong on the map, i will ask Antoine about it

Reply:
Done the changes